examples/06-plotting/08_plot_3d_streamlines.py

```diff
@@ -0,0 +1,171 @@
+"""
+.. _plot_3d_streamlines:
+
+Plot 3D streamlines
+~~~~~~~~~~~~~~~~~~~
+This example shows you how to plot streamlines of fluid simulation results,
+for 3D models.
+
+"""
+
+###############################################################################
+# Plot streamlines from single source
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+###############################################################################
+# Import modules, create the data sources and the model
+# -----------------------------------------------------
+# Import modules:
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+from ansys.dpf.core.plotter import DpfPlotter
+
+###############################################################################
+# Create data sources for fluids simulation result:
+fluent_files = examples.download_fluent_mixing_elbow_steady_state()
+ds_fluent = dpf.DataSources()
+ds_fluent.set_result_file_path(fluent_files["cas"][0], "cas")
+ds_fluent.add_file_path(fluent_files["dat"][1], "dat")
+
+###############################################################################
+# Create model from fluid simulation result data sources:
+m_fluent = dpf.Model(ds_fluent)
+
+###############################################################################
+# Get meshed region and velocity data
+# -----------------------------------
+# Meshed region is used as geometric base to compute the streamlines.
+# Velocity data is used to compute the streamlines. The velocity data must be nodal.
+
+# Get the meshed region:
+meshed_region = m_fluent.metadata.meshed_region
+
+# Get the velocity result at nodes:
+velocity_op = m_fluent.results.velocity()
+fc = velocity_op.outputs.fields_container()
+field = dpf.operators.averaging.to_nodal_fc(fields_container=fc).outputs.fields_container()[0]
+
+###############################################################################
+# Plot the streamlines adjusting the request
+# ------------------------------------------
+# The following steps show you how to create streamlines using DpfPlotter, with several sets
+# of parameters. It demonstrates the issues that can happen and the adjustments that you can make.
+
+# First, a DpfPlotter is created and the streamline is created with default values.
+pl0 = DpfPlotter()
+try:
+    pl0.add_mesh(meshed_region=meshed_region, opacity=0.3)
+    pl0.add_streamlines(meshed_region=meshed_region, field=field)
```

Comment:
@anslpa is the goal of this feature to plot streamline or to compute streamlines? We will not be able to retrieve the actual data ?

Reply:
Create fluid_helpers for streamline computation, either create a function or a class that would look like an operator. This method would be called "fluid_helpers.compute_streamline"

Reply:
Ideal would be to hav ea streamline per field in the fields container ? How the data is containerized per streamline in PolyData ?

Reply:
# PyDpf-Core
## Compute streamline and then plot it
streamlines_fc_3D = dpf.helpers.compute_streamline(meshed_region, field=velocity_field, source_center=(...), source_radius=0.001)
streamlines_fc_2D = dpf.helpers.compute_streamline(meshed_region=meshed_region, field=field, return_source=True, radius=0.00002, start_position=(0.005, 0.0005, 0.0), surface_streamlines=True)
pl2 = DpfPlotter()
pl2.add_streamlines(computed_streamline=streamlines_fc_3D, radius=0.001)
pl2.show_figure(show_axes=True)

## Plot streamline

```
pl2 = DpfPlotter()
pl2.add_streamlines(
    meshed_region=meshed_region,
    field=field,
    return_source=True,
    source_center=(0.56, 0.48, 0.0),
    radius=0.001,
    n_points=10,
    source_radius=0.075,
    max_time=10.0,
)
pl2.show_figure(show_axes=True)
```

# PyDpf-Post

Reply:
@anslpa yes with https://docs.pyvista.org/version/stable/api/core/_autosummary/pyvista.DataSetFilters.streamlines_from_source.html#pyvista.DataSetFilters.streamlines_from_source
So the method taking in a PyDPF-Core geometry object would need to translate it to a `pv.DataSet`

Reply:
quid "return_source" ? source as field ?

Reply:
Handled in this PR: https://github.com/ansys/pydpf-core/pull/1062/files.